src/main/resources/data/spectrum/advancements/unlocks/items/bag_of_holding.json

```diff
@@ -1,20 +1,37 @@
 {
   "criteria": {
     "placed_pedestal": {
-      "trigger":"revelationary:advancement_gotten",
+      "trigger": "revelationary:advancement_gotten",
       "conditions": {
         "advancement_identifier": "spectrum:place_pedestal"
       }
     },
     "collected_ender_eye": {
-      "trigger":"revelationary:advancement_gotten",
+      "trigger": "revelationary:advancement_gotten",
       "conditions": {
         "advancement_identifier": "spectrum:hidden/collect_vanilla/ender_eye"
       }
     },
+    "collected_warp_flux": {
+      "trigger": "minecraft:inventory_changed",
+      "conditions": {
+        "items": [
+          {
+            "items": [
+              "malum:warp_flux"
```

Comment:
doesn't this throw an error when malum is not loaded, making the whole file break?

Reply:
.... oh dear yeah it does. how would I fix this then?

Reply:
Fixed